src/libraries/Native/build-native.cmd

```diff
@@ -145,7 +145,7 @@ goto :Failure
 
 :BuildNativeProj
 :: Build the project created by Cmake
-set __msbuildArgs=/p:Platform=%__BuildArch% /p:PlatformToolset="%__PlatformToolset%"
+set __msbuildArgs=/p:Platform=%__BuildArch% /p:PlatformToolset="%__PlatformToolset%" -noWarn:MSB8065
```

Comment:
Nit: Link to the issue to get this workaround removed would be nice

Reply:
Thanks. We don’t have an issue open to update the toolset in the build pools. I’m going to merge and once I’m back from vacation will follow up on this matter and add the issue link if needed.